search-service/src/test/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeServiceTests.kt

```diff
@@ -317,6 +317,7 @@ class TemporalEntityAttributeServiceTests : WithTimescaleContainer {
                     incomingAttrExpandedName,
                     outgoingAttrExpandedName
                 )
+
```

Comment:
remove this modification, it is the only in this file and creates un-necessary noise